src/ui/src/components/core/content/CoreDataframe/CoreDataframeCellText.vue

```diff
@@ -21,6 +21,38 @@ function stopEditing() {
 	if (newValue === props.value) return;
 	emits("change", textarea.value.value);
 }
+
+function autoResizeFromRef() {
+	if (!textarea.value?.ref) return;
+
+	const el = textarea.value.ref as HTMLTextAreaElement;
+
+	const style = getComputedStyle(el);
+	if (!style) return;
+
+	el.style.height = "auto";
+
+	const borderTop = parseFloat(style.borderTopWidth || "0");
+	const borderBottom = parseFloat(style.borderBottomWidth || "0");
```

Comment:
1. I guess it's better to avoid `NaN` this way
2. we can use integer for px
```suggestion
	const borderTop = parseInt(style.borderTopWidth) || 0;
	const borderBottom = parseInt(style.borderBottomWidth) || 0;
```